tools/common/common-build.props

```diff
@@ -9,8 +9,10 @@
     <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)'==''">10.0.14393.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion Condition="'$(WindowsTargetPlatformMinVersion)'==''">10.0.10586.0</WindowsTargetPlatformMinVersion>
     <EnableDotNetNativeCompatibleProfile Condition="'$(EnableDotNetNativeCompatibleProfile)'==''">true</EnableDotNetNativeCompatibleProfile>
+    <InferPackageContents>false</InferPackageContents> <!-- DH: Exploratory fix for Nugetizer-3000 0.2.0 changes that cause DUPLICATE FILE errors. -->
     <IncludeOutputsInPackage>false</IncludeOutputsInPackage>
```

Comment:
my "exploratory fix" comments should probably be purged

Reply:
With this fix, can you produce a WinObjC.Logging.nupkg? If so, how big is it?

---
In reply to: [141440908](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141440908) [](ancestors = 141440908)